tests/functional/api/test_objectstorage.py

```diff
@@ -1087,6 +1087,11 @@ def test_container_snapshot(self):
         # Non existing snapshot should work
         self.api.container_snapshot(self.account, container, self.account,
                                     snapshot)
+        # Check sys.user.name is correct
+        ret = self.api.container_get_properties(self.account, snapshot)
+        self.assertEqual(ret['system']['sys.user.name'], snapshot)
+        self.assertEqual(ret['system']['sys.account'], self.account)
```

Comment:
The assertions are supposed to be done in the reverse order: *expected* before *actual*. Some assertion methods do not specify it, other (like `assertDictEqual`) do and produce better error messages is the order is respected.